ncm-grub/src/main/perl/grub.pm

```diff
@@ -191,11 +193,25 @@ sub password
     my $val = $tree->{option} ? "--$tree->{option} " : "";
     $val .= $password;
 
-    $grub_fh->add_or_replace_lines(qr/^password\s+/,
-                                   qr/^password $val$/,
-                                   "password $val\n",
-                                   $self->main_section_offset($grub_fh),
-                                   );
+    if ( -d $GRUB2_DIR ) {
```

Comment:
I get that, but my point is that /etc/grub.d/01_users seems to be common across UEFI and BIOS as part of grub2-tools, so wherever the grub.cfg is it should contain the same block which reads $prefix/user.cfg. And /boot/grub2 will be there for both too.